### ISSUE TITLE
docker image deployment fails to run cluster serving demo code

### ISSUE THREAD

**Issue by wan9guan** [intel-analytics/analytics-zoo#3421]
Hello team,
I downloaded analytics zoo docker image from your team and offline set up my single node server.  But I failed to run cluster serving demo code. this is demo code is provided by your team in a medical image inference POC.  Please help.

when do below:
$FLINK_HOME/bin/start-cluster.sh
it returns empty value.

when do below:
$FLINK_HOME/bin/flink list
it shows result as attached picture (1.jpg) .
![1](https://user-images.githubusercontent.com/77830100/105467573-d9164380-5cd0-11eb-9f50-cf4d8c7bf8fe.jpg)

Please help. Thanks.

**Comment by Litchilitchy** [comment]
please post your python code and `$FLINK_HOME/log/flink-xxx-taskexecutor.log`

Use text instead of photo would be better

**Comment by wan9guan** [comment]
python code is below:
======================== ====
# from segmentation_models import Xnet
# from tensorflow import keras
# import tensorflow as tf

import cv2
import numpy as np
# import keras
import SimpleITK as sitk
from zoo.serving.client import InputQueue, OutputQueue

# Use following code to transform Keras h5 to TF savedModel
# model = tf.keras.models.load_model("./1.h5")
# tf.saved_model.save(model, "tf_seg")


s = 210
filename = './volume-124.nii'
itkimage = sitk.ReadImage(filename)
images = sitk.GetArrayFromImage(itkimage).astype(np.float32)
images = np.clip(images, -135, 215)
images = (images + 135)/350.
images = np.transpose(images,(1,2,0))
print(images.shape)
gname = './segmentation-124.nii'
itkimage = sitk.ReadImage(gname)
g = sitk.GetArrayFromImage(itkimage).astype(np.float32)

input = images[:,:,s-1:s+2]
input1 = images[:,:,s:s+3]
print(input.shape)
# input = np.expand_dims(images[:,:,s-1:s+2],axis=0)
input_api = InputQueue()

input_api.enqueue('my-image1', t=input)
input_api.enqueue('my-image2', t=input1)
# This is a demo to show result, so we wait 10 sec to ensure result exists
# In production, no sleep is needed
import time
time.sleep(10)
start = time.time()
output_api = OutputQueue()
p = output_api.query('my-image1')
print(time.time()-start)
start = time.time()
output_api = OutputQueue()
p = output_api.query('my-image2')
print(time.time()-start)
start = time.time()
output_api = OutputQueue()
p = output_api.query('my-image1')
print(p)
print(time.time()-start)

# p = model2.predict(input,batch_size = 1)[0]
#cv2.imshow('p',p/2)
#cv2.imshow('g',g[s]/2)
#cv2.imshow('o',images[:,:,s-1:s+2])

**Comment by wan9guan** [comment]
Please see log file as attached.
[flink--client-a18c1d7c8240.zip](https://github.com/intel-analytics/analytics-zoo/files/5854824/flink--client-a18c1d7c8240.zip)

**Comment by gtfaiwxm** [comment]
Have you modified the config.yaml, when you use cluster serving inference segmentation model?

**Comment by Litchilitchy** [comment]
@wan9guan from log, Cluster Serving started successfully but did not get any input. But the python client said it has written to Redis. 

So could you `$REDIS_HOME/src/redis-cli` and `xlen serving_stream` and post the output? This is to check if the inputs really exist in Redis.

**Comment by wan9guan** [comment]
Yes I modified config.yaml.      Model directory changed to    /opt/work/AZ/tf_seg

**Comment by wan9guan** [comment]
Dear,  ok let me try to dump out the output.      Meanwhile, is it possible you remote log onto my server to quick check ?

**Comment by wan9guan** [comment]
after running the command, output is: 
127.0.0.1:6379> xlen serving_stream
(integer) 2
127.0.0.1:6379>

**Comment by wan9guan** [comment]
远程访问：
首先用浏览器直接访问https://60.16.5.156:7443
然后直接选择右边的HTML ACCESS
然后输入VMware用户名qiuwx，密码1qaz!QAZ
然后就能看到那个LinkClone-Remote，点进去
这时应该就能看到centos的控制台。你也可以用putty再建立一个连接。点击putty，输入ip 10.170.26.35
然后输入用户名root，密码aaa123+-*/

**Comment by Litchilitchy** [comment]
Problem solved after our offline sync.

When Cluster Serving give empty output, or does not work, you could always check it according to [dubug tutorial])(https://github.com/intel-analytics/analytics-zoo/blob/master/docs/docs/ClusterServingGuide/DebugGuide.md)
